### PR TITLE
marcel soln

### PR DESCRIPTION
**Pull request by marcelkooi**

### CODE REVIEW

**Comment by paradime** [comment]
:+1:

File: word_counter.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+def word_count(input_file)
+
+	text = IO.read(input_file).gsub(/\W/, ' ').split(' ')
```
Comment (jaybobo) [inline word_counter.rb]: nitpick: 2 spaces ;)

Comment (marz0) [inline word_counter.rb]: It's set to 2 spaces on my text editor, is it a github setting to change?

Comment (austenmadden) [inline word_counter.rb]: If you are using sublime there is another line you have to add beyond indent size:
"translate_tabs_to_spaces": true

If you are using another text editor there is probably a similar setting somewhere.

Comment (marz0) [inline word_counter.rb]: Thanks!

**Comment by jaybobo** [comment]
:+1:

**Pull request closed without merging**